### PR TITLE
feat(rbac): add api_usages api resource

### PR DESCRIPTION
**Pull request by guangrui-cloudnatix**

### CODE REVIEW

File: deployments/rbac-server/values.yaml [hunk 1]
```diff
@@ -74,10 +78,12 @@ roleScopesMap:
   - api.batch.jobs.write
   - api.files.read
   - api.files.write
-  - api.vector-stores.read
-  - api.vector-stores.write
+  - api.vector_stores.read
```
Comment (kkaneda) [inline deployments/rbac-server/values.yaml:81]: Do you plan to update https://github.com/llmariner/vector-store-manager/blob/main/server/internal/server/server.go#L97 ?

Comment (guangrui-cloudnatix) [reply deployments/rbac-server/values.yaml:81]: question: does it need to match with the resource path defined in `proto` and `ingress`?

Comment (kkaneda) [reply deployments/rbac-server/values.yaml:81]: @guangrui-cloudnatix , hm, I think we don't have any specific convention. Changing to `vector_stores` looks more consistent with the rest, but we just need to make the corresponding change to `vector-store-manager`.

**Review by kkaneda — APPROVED** [review]

File: deployments/rbac-server/values.yaml (hunk 1 above)
Comment (guangrui-cloudnatix) [reply deployments/rbac-server/values.yaml]: I see. revert the changes for `vector-stores`

**Pull request merged**